src/redux/books/books.js

```diff
@@ -1,47 +1,82 @@
-import { v4 as uuidv4 } from 'uuid';
+// import { v4 as uuidv4 } from 'uuid';
```

Comment:
- [ ] Unnecessary commented-out codes should be removed from the codebase to make it look clean and smart. This is among the coding best practices. So please remove this line of code 😃 👍